skglm/experimental/tests/test_quantile_huber.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from numpy.testing import assert_allclose
+from sklearn.linear_model import QuantileRegressor
+from sklearn.datasets import make_regression
+from skglm.experimental.quantile_huber import SmoothQuantileRegressor
+
+
+@pytest.mark.parametrize('quantile', [0.3, 0.5, 0.7])
```

Comment:
@floriankozikowski also parametrize the test to use fit_intercept=True and False